src/js/APIHelper.js

```diff
@@ -0,0 +1,27 @@
+export default class APIHelper {
+  static baseURL = 'https://api.tvmaze.com/shows';
+
+  static async getAll(page = 1) {
+    try {
+      const shows = await fetch(`${APIHelper.baseURL}?page=${page}`).then(
+        (response) => response.json(),
+      );
+      return shows;
+    } catch {
+      console.log("Error: Couldn't fetch films data");
```

Comment:
[OPTIONAL] Good job catching errors when doing requests to API's. It is also a good practice to not just console.log these errors, instead displaying a message error on the application with feedback for the user leads to a better user experience.